<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDlg</class>
 <widget class="QDialog" name="loginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>489</y>
     <width>260</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>159</y>
     <width>812</width>
     <height>487</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>resource/login/dlg.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>179</y>
     <width>448</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>resource/login/logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>319</y>
     <width>110</width>
     <height>109</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>resource/login/head.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>489</y>
     <width>260</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_developer">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>210</y>
     <width>83</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>83</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>开发者</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_admin">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>210</y>
     <width>83</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>83</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tester">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>210</y>
     <width>83</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>83</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>测试员</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>320</y>
     <width>256</width>
     <height>37</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>37</height>
    </size>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>gy</string>
   </property>
   <property name="maxLength">
    <number>128</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>390</y>
     <width>256</width>
     <height>37</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>37</height>
    </size>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>gy17305</string>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>29</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>29</width>
     <height>29</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>29</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>resource/login/icon1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>320</y>
     <width>29</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>29</width>
     <height>29</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>29</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>resource/login/icon3.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>319</y>
     <width>10</width>
     <height>109</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>109</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>2</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>resource/login/split.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_copyright">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>700</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Copyright(C)2015 NingBo Textile Instrument Factory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_account">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>445</y>
     <width>256</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_login</zorder>
  <zorder>pushButton_login</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_cancel</zorder>
  <zorder>pushButton_developer</zorder>
  <zorder>pushButton_admin</zorder>
  <zorder>pushButton_tester</zorder>
  <zorder>lineEdit_name</zorder>
  <zorder>lineEdit_password</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_copyright</zorder>
  <zorder>label_account</zorder>
  <zorder>label_message</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
